pkg/azclient/policy/retry.go

```diff
@@ -113,7 +110,7 @@ func (p *GetRequestRegionalEndpointRetryPolicy) Do(req *policy.Request) (*http.R
 
 	bodyString := string(bodyBytes)
 	trimmed := strings.TrimSpace(bodyString)
-	klog.V(5).Infof("Send.sendRequest got response with ContentLength %d, StatusCode %d and responseBody length %d", response.ContentLength, response.StatusCode, len(trimmed))
+	//klog.V(5).Infof("Send.sendRequest got response with ContentLength %d, StatusCode %d and responseBody length %d", response.ContentLength, response.StatusCode, len(trimmed))
```

Comment:
Could you delete it?

Reply:
retry with regional endpoint is not enabled by default. we  need to enabled it when the config parser is added. We also need to unified logger, maybe we can add them back later?

Reply:
code has been deleted.